packages/cozy-doctypes/src/File.js

```diff
@@ -125,6 +125,29 @@ class CozyFile extends Document {
       throw error
     }
   }
+  /**
+   * Method to generate a new filename if there is a conflict
+   *
+   * @param {String} filenameWithoutExtension A filename without the extension
+   * @return {String} A filename with the right suffix
+   */
+  static generateNewFileNameOnConflict(filenameWithoutExtension) {
```

Comment:
This means that if you want to save "hello.png" and there is a "hello.doc" you willl end to save a "hello_1.png". Is this really what we want ?

Reply:
If you call this method where there is no name conflict, I can't do anything for you... 

This method should only be called when `statByPath` returns a something. 

Actually this `generateNewFileNameOnConflict` is the very low method, higher function will come pretty soon